docs/paper/joss.md

```diff
@@ -0,0 +1,263 @@
+
+With JustRelax.jl, we intend to address these fundamental limitations by introducing the first HPC-ready geodynamic software written in Julia:
+
+- We leverage Julia's metaprogramming capabilities to generate hardware-specific code. With this, any script written by the user runs efficiently across different architectures and hardware with minimal changes.
+
+- We use a solution method which is particularly well-suited for GPU accelerators. 
+
+- Julia's built-in package manager makes the installation of JustRelax.jl trivial, as it automatically downloads and installs all the required dependencies for any compatible operating system (Windows, macOs, and most of the mainstream Linux distros).
+
+- Julia is a higher-level language, with a syntax that may be familiar to Python and MATLAB users. This significantly lowers the entry bar to new users, while still offering a performance comparable to static languages.
+
+# Methods
+
+As previously stated, JustRelax.jl employs the APT method to solve the compressible Stokes and heat diffusion equations @rass2022. Within each APT iteration, the pressure-dependent equation of state, non-Newtonian viscosity, plastic stress corrections, and conservation of mass and momentum are solved simultaneously in a fully coupled manner. In the sections below, we briefly summarize the governing equations and their respective APT implementation, and discuss the parallelization strategy and the advection schemes used to track material properties and deformation history.
+
+## Thermo-mechanical modeling
+The compressible Stokes equations  are given by the conservation of momentum and mass equations, respectively:
+$$
+\begin{align}
+    \nabla\cdot\mathbf{\tau} - \nabla p = \mathbf{f} \\
+    \nabla\cdot\mathbf{v} = -\beta \frac{\partial p}{\partial t} + \alpha \frac{\partial T}{\partial t}
+\end{align}
+$$
+
+where $\nabla$ is the nabla operator, $\mathbf{\tau}$ is the deviatoric stress tensor, $p$ is the total pressure, $\mathbf{f}$ are the body forces, usually being the buoyancy forces $\mathbf{f} = \rho \mathbf{g}$ where $\rho$ is density and $\mathbf{g} = (0, -g_y)$ is the gravitational force vector, $\mathbf{v}$ is the velocity field, $\beta$ is the inverse of the bulk modulus, and $\alpha$ is the thermal expansivity coefficient. In the incompressible limit, the right-hand-side of the conservation of mass equation vanishes to zero. The previous system of equations is closed with the constitutive equation that relates the deviatoric strain tensor $\dot{\mathbf{\varepsilon}}$ with $\mathbf{\tau}$. In computational geodynamics, we commonly consider composite non-linear visco-elasto-plastic rheologies, which can be expressed by the following constitutive equation:
```

Comment:
$\mathbf{g} = (0, -g_y)$ suggests a 2D setup, while the equations are valid for any number of dimensions here. I would just state the g is gravity, but leave it also dimension agnostic.